<launch>
    <group ns="dodobot" >
        <node name="db_bbox_to_pose" pkg="db_bbox_to_pose" type="db_bbox_to_pose.py" output="screen">
            <param name="depth_topic"           value="/camera/aligned_depth_to_color/image_raw"/>
            <param name="camera_info_topic"     value="/camera/color/camera_info"/>
            <param name="depth_info_topic"      value="/camera/aligned_depth_to_color/camera_info"/>
            <param name="detect_topic"          value="/detectnet/detections"/>
            <param name="publish_debug_image"     value="true"/>

            <rosparam param="labels">
                - BACKGROUND
                - cozmo_cube
                - blue_cut_sphere
                - red_cut_sphere
                - blue_low_bin
                - red_low_bin
                - blue_cube
                - red_cube
            </rosparam>

            <rosparam param="marker_colors">
                BACKGROUND: [0.0, 0.0, 0.0, 1.0]
                cozmo_cube: [0.95, 1.0, 1.0, 1.0]
                blue_cut_sphere: [0.14, 0.75, 1.0, 1.0]
                red_cut_sphere: [1.0, 0.2, 0.13, 1.0]
                blue_low_bin: [0.13, 0.24, 0.52, 1.0]
                red_low_bin: [0.52, 0.2, 0.14, 1.0]
                blue_cube: [0.14, 0.75, 1.0, 1.0]
                red_cube: [1.0, 0.2, 0.13, 1.0]
            </rosparam>

        </node>
    </group>
</launch>
